lib/trento/application/integration/discovery/protocol/enrich_request_host_deregistration.ex

```diff
@@ -17,32 +22,41 @@ defimpl Trento.Support.Middleware.Enrichable,
           | {:error, :host_alive}
           | {:error, :host_not_registered}
   def enrich(%RequestHostDeregistration{host_id: host_id} = command, _) do
-    Repo.get(HostReadModel, host_id)
-    |> Repo.preload([:heartbeat_timestamp])
+    HostReadModel
+    |> where([h], h.id == ^host_id)
```

Comment:
Should we check here if the host was already deregistered?
Or are we going to "reject" these deregistration commands in the domain aggregate itself.

Reply:
We are already checking if the host is already deregistered in the `host_deregisterable()` function.

Reply:
(There is also a test for this)